recipe/meta.yaml

```diff
@@ -1,45 +1,51 @@
-{% set name = "<PLEASE ADD PKG NAME>" %}
-{% set version = "<PLEASE ADD PKG VERSION>" %}
+{% set name = "streamlit-mermaid" %}
+{% set version = "0.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: <sha256>
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/streamlit-mermaid-{{ version }}.tar.gz
+    sha256: a7883f63be991a378b2c327efae9a03ae49e830481659ae1a955bf68597f9c48
+  # License was not included in tarball, pulling from github for now
+  # Remove this when upstream includes it in pypi release
+  - url: https://raw.githubusercontent.com/neka-nat/streamlit-mermaid/refs/heads/main/LICENSE
+    sha256: f11403a5951d5e69fc315374ca67147a48a7c9c36dbfe6512f59a78394473497
 
 build:
-  number: 0
+  skip: True  # [py<310 or s390x]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  skip: True  # [py<38]
+  number: 0
 
 requirements:
   host:
     - python
+    - poetry-core
     - pip
   run:
     - python
+    - streamlit >=1.0
```

Comment:
`pyproject.toml` and `setup.py` both say `streamlit >= 1.24.1` ??

Reply:
Wait, lol, the pypi tarball and git repo are different...

Reply:
This differs between tarball and github, do you think we should go with higher version just to be safe?

Reply:
If we could ping the authors and ask "why so?" that might be good.
Technically, as we're building from the pypi tarball, I'd follow its dependencies.
The git repo hasn't had those files touched, even if they've made other changes (like the LlCENSE file 😉 ).

Hmm, even the 0.1.0 pypi tarball has the higher dependency.

Plus, if it builds and tests within our ecosystem, then we should be OK.

Reply:
I've updated the version to be as in tarball